Sazabi.rc

```diff
@@ -1893,8 +1893,8 @@ END
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION 12,111,96,2
- PRODUCTVERSION 12,111,96,2
+ FILEVERSION 12,111,98,1
+ PRODUCTVERSION 12,111,98,1
  FILEFLAGSMASK 0x3fL

```

Comment:
It seems that PRODUCTVERSION is not consistent in Sazabi.rc
For example, 12,111,98,3 and 12,111,98,1 co-exists.